cli_docs/app.md

```diff
@@ -0,0 +1,979 @@
+# `aerie-cli app`
+
+**Usage**:
+
+```console
+$ aerie-cli app [OPTIONS] COMMAND [ARGS]...
```

Comment:
Is there a way to update the configuration to make these docs show all these commands at the root, i.e., `aerie-cli [COMMAND]` instead of `aerie-cli app [COMMAND]`?